templates/inactive_user_email.html

```diff
@@ -0,0 +1,20 @@
+<p>
+  You are receiving this notification because you are registered as a user for
+  WebStatus and have been inactive for longer than 6 months.
+</p>
+</br>
+<p>
+  If you still need the ability to create features on WebStatus, logging into
+  <a href="{{site_url}}">{{site_url}}</a> with this email will avoid any
+  disruption in your access.
+</p>
+</br>
+<p>
+  For the sake of site security, a user with 9 months of inactivity is
+  granted normal user access. If you no longer need special access, no
+  action is required on your part.
+  Please note that you if you need special access again, you may contact us at
+  webstatus-request@google.com to re-establish special access.
+</p>
+</br>
+<p>(this is an automated message generated by the WebStatus team)</p>
```

Comment:
this -> This
and
WebStatus -> ChromeStatus.com

Actually, I think we could do without this line.  The message will have a From: line that says "Chromestatus admin@cr-status.appspotmail.com", and our users are pretty sophisticated, so I think they will know that it is an automated email.

Reply:
I see - I've removed this last line as well as making the other suggestion changes 👍